lib/engine/game/g_18_eu/step/modified_dutch_auction.rb

```diff
@@ -91,6 +91,9 @@ def pass_auction(entity)
 
           def remove_from_auction(entity)
             @active_bidders.delete(entity)
+            @bids[@auctioning]&.reject! do |bid|
+              bid.entity == entity
+            end
```

Comment:
Without this change, bids would just linger in the UI until the auction was concluded.